code/modules/reagents/withdrawal/generic_addictions.dm

```diff
@@ -21,6 +21,7 @@
 /datum/addiction/opiods/end_withdrawal(mob/living/carbon/affected_carbon)
 	. = ..()
 	affected_carbon.remove_status_effect(STATUS_EFFECT_HIGHBLOODPRESSURE)
+	affected_carbon.set_disgust(0)
```

Comment:
this kills any disgust you might've had beforehand, and should not be done like this.

Reply:
I mean, getting addicted to opiods and then withdrawing seems too convoluted for the average ss13 player. I'll just reduce it to a sane amount

Reply:
It now just halves their disgust